game/scripts/npc/abilities/lina_dragon_slave.txt

```diff
@@ -27,7 +27,7 @@
 
     // Damage.
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityDamage"                                       "85 160 235 310 660 1060"
+    "AbilityDamage"                                       "120 200 280 360 700 1100" //OAA
```

Comment:
I think Dragon Slave lvl 5 and 6 don't need a buff.

Reply:
Why though, this and death prophets Q are basically the same spell after talents.

Reply:
Lina has a spell amp talent, DP doesnt.